tests/unit/mixins/adapter-fetch-test.js

```diff
@@ -246,6 +246,28 @@ test('mungOptionsForFetch sets the request body correctly when the method is POS
   assert.deepEqual(options.body, stringifiedData);
 });
 
+test('mungOptionsForFetch does not process an empty \'data\' object', function(assert) {
```

Comment:
https://en.wikipedia.org/wiki/Mung_(computer_term)

Guess it can be spelled both ways? I've always had `mung` in this repo.